### DISCUSSION TITLE
Maintenance Recs - Getting started with my new QQ-S Pro

### DISCUSSION THREAD

**Discussion by justinmreina** [blackfyre/flsun.community#24] [General]
Getting started with my new QQ-S Pro, in with the mail today and very excited. Cartesian had been high investments for me, first w/a Creality Ender 5 Pro and then a Prusa Mini+ always needing maintenance or adjustment; these things took a lot of effort.

I switched to the QQ-S Pro to alleviate this, I spent more time and money on maintenance than I did on prints, was frustrating. So here I ask - 

&nbsp;

**Maintenance Check**
- What maintenance and best practice recommendations are there for use with the QQ-S Pro? Let's assume I have 75% uptime using OctoPrint, changing inks many times per week

**Comment by blackfyre** [comment]
The QQ-S Pro is relatively maintenance free... apart from keeping it from collecting dust and preventing buildup of it is your most important thing to do... apart from occasionally checking the ptfe tubing for damages...
I'd recommend stocking up on:
 * thermistors
 * heat cartridges
 * nozzles
 * ptfe tubing
 * heat blocks
 * timing belts
 * lithium grease for the fisheye bearings

Another modification could be the replacement of the solder joints above the effector to 2 pin inline jst connectors.
